shared/src/commonMain/kotlin/org/hyperskill/app/stages/injection/StagesDataComponent.kt

```diff
@@ -0,0 +1,9 @@
+package org.hyperskill.app.stages.injection
+
+import org.hyperskill.app.stages.domain.interactor.StagesInteractor
+import org.hyperskill.app.stages.domain.repository.StagesRepository
+
+interface StagesDataComponent {
+    val stagesRepository: StagesRepository
```

Comment:
Why should repository be exposed for component users?

Reply:
I thought the relations between repository and interactor is 1-to-1.

Reply:
@XanderZhu we can discuss this later, architecture -> https://hackmd.io/@eadm/BJg82eA7N. Maybe I didn't understand something correctly.